docs/content/development/extensions-core/lookups-cached-global.md

```diff
@@ -28,7 +28,6 @@ Cached namespace lookups can be specified as part of the [cluster wide config fo
     "type": "cachedNamespace",
     "extractionNamespace": {
        "type": "uri",
-       "namespace": "some_uri_lookup",
```

Comment:
If it's not required then lookups-cached-global.md should be updated too, as it does say "namespace" is required